cli/src/cmd_net.rs

```diff
@@ -604,29 +893,29 @@ pub struct CmdBgpPeer {
 impl AuthenticatedCmd for CmdBgpPeer {
     async fn run(&self, client: &Client) -> Result<()> {
         match &self.subcmd {
-            BgpConfigPeerSubCommand::Add(cmd) => cmd.run(client).await,
-            BgpConfigPeerSubCommand::Del(cmd) => cmd.run(client).await,
+            BgpConfigPeerSubCommand::Set(cmd) => cmd.run(client).await,
+            BgpConfigPeerSubCommand::Delete(cmd) => cmd.run(client).await,
         }
     }
 }
 
 #[derive(Parser, Debug, Clone)]
 enum BgpConfigPeerSubCommand {
-    /// Add a BGP peer to a port configuration.
-    Add(CmdBgpPeerAdd),
+    /// Set a BGP peer to a port configuration.
+    Set(CmdBgpPeerSet),
 
     /// Remove a BGP from a port configuration.
-    Del(CmdBgpPeerDel),
+    Delete(CmdBgpPeerDel),
 }
 
-/// Add a BGP peer to a given switch port configuration.
+/// Set a BGP peer configuration.
 ///
 /// This performs a read-modify-write on the specified switch port
 /// configuration.
 #[derive(Parser, Debug, Clone)]
 #[command(verbatim_doc_comment)]
-#[command(name = "add")]
-pub struct CmdBgpPeerAdd {
+#[command(name = "set")]
+pub struct CmdBgpPeerSet {
     /// Id of the rack to add the peer to.
```

Comment:
All of the help strings in this section currently say "add", but should be updated to the new set semantics, perhaps "configure", since that is the wording used for `oxide system networking route set`?

Reply:
changed add to set and adjusted grammar accordingly